setup.cfg

```diff
@@ -0,0 +1,35 @@
+# Changes required for PyPI
+[metadata]:
+name = vcf2hl7v2
+version = 0.0.4
```

Comment:
```suggestion
version = 0.0.1
```

Reply:
Actually, while deploying on test `PyPI`, I faced some errors when installing it via pip due to which I had to deploy 3 more times changing the version every time. The version that is working correctly is [`0.0.4`](https://test.pypi.org/project/vcf2hl7v2/).